src/drafts/SelectPanel2/SelectPanel.tsx

```diff
@@ -250,6 +250,7 @@ const Panel: React.FC<SelectPanelProps> = ({
             '100%': {transform: 'scale(1, 1)'},
           },
         }}
+        {...props}
```

Comment:
@siddharthkp Does the typescript on `props` also allow `StyledOverlay` props to be passed through? 

For context, I'm asking case where the prop `maxHeight` is needed to fix a styling issue with the overlay height: https://github.com/github/collaboration-workflows-flex/issues/449
```
<SelectPanel
      height="fit-content"
      maxHeight="large"
      ...
/>
```

Reply:
>  Does the typescript on props also allow StyledOverlay props to be passed through?

Not yet!

> I'm asking case where the prop maxHeight is needed to fix a styling issue with the overlay height: https://github.com/github/collaboration-workflows-flex/issues/449

That's an interesting issue, not sure if maxHeight is the solution. Going to reply on the issue first itself